srcpkgs/nautilus/template

```diff
@@ -18,9 +18,9 @@ short_desc="GNOME file manager"
 maintainer="Michal Vasilek <michal@vasilek.cz>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
-changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-43/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/44.0/NEWS"
 distfiles="${GNOME_SITE}/nautilus/${version%%.*}/nautilus-${version}.tar.xz"
-checksum=a86a9aa7a455511b029ce49a1d8b4659c3c355b5d81d7360bb4d0de637afede0
+checksum=57b99ebb8e2b9c1512d381e53096188c087b33410d6c52d87a27bd60ee76ac7f
 # tests timeout:
 # 11/11 test-nautilus-search-engine-tracker            TIMEOUT        480.00s
 make_check=extended
```

Comment:
this test takes 0.4s for me, can we remove this?

Reply:
CI fails without it, local build works fine

Reply:
then we should use ci-skip instead of extended